app/api/schema/event_role_permissions.py

```diff
@@ -0,0 +1,37 @@
+from marshmallow_jsonapi import fields
+from marshmallow_jsonapi.flask import Schema, Relationship
+
+from app.api.helpers.utilities import dasherize
+
+
+class EventsRolePermissionSchema(Schema):
```

Comment:
When you've defined ```default``` values for some fields in the Schema, please use the ```use_defaults``` decorator. It's declared in the ```utils/common.py``` file. If you don't use it, then it won't add the default values before schema validation.

Reply:
Done @srv-twry Please review.